pyproject.toml

```diff
@@ -11,15 +11,16 @@ homepage = "https://github.com/Shard-Labs/starknet-devnet"
 keywords = ["starknet", "cairo", "testnet", "local", "server"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.7.2"
```

Comment:
Why?

Reply:
Because

Reply:
`ipfshttpclient` package requires python version to be greater than 3.7.1